app/upgrades/v16/upgrades.go

```diff
@@ -1,15 +1,59 @@
 package v16
 
 import (
+	ratelimitkeeper "github.com/Stride-Labs/ibc-rate-limiting/ratelimit/keeper"
+	ratelimittypes "github.com/Stride-Labs/ibc-rate-limiting/ratelimit/types"
+
 	icacontrollertypes "github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/controller/types"
 
+	errorsmod "cosmossdk.io/errors"
+	sdkmath "cosmossdk.io/math"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
 
 	"github.com/cosmos/gaia/v16/app/keepers"
 )
 
+var RateLimits = map[string]ratelimittypes.MsgAddRateLimit{
+	"osmosis-1": {
+		MaxPercentSend: sdkmath.NewInt(5),
+		MaxPercentRecv: sdkmath.NewInt(5),
+		ChannelId:      "channel-141",
```

Comment:
I guess `ChannelId`s are taken from the proposal but how were those initially found?
From what I see [here](https://ping.pub/cosmos/ibc/connection/connection-257) we have multiple channels under `transfer` between the Hub and Osmosis. Might be nice to have a comment on this.

Reply:
Creating transfer channels is permissionless. This means that there can be multiple channels between two chains. However, social consensus chooses one as the canonical transfer channel. In the case of Osmosis, this is channel-141 -- the channel with all the IBC transfers (see https://www.mintscan.io/cosmos/relayers). Another way to look at this is to check the assets on the Hub https://www.mintscan.io/cosmos/assets. I don't think adding a comment with a link helps as this link might change in the future.